docs/src/main/paradox/ironmq.md

```diff
@@ -0,0 +1,76 @@
+# IronMq Connector
+
+The IronMq connector provides an Akka stream source and sink to connect to the [IronMQ](https://www.iron.io/platform/ironmq/) queue.
+
+IronMq is a simple point-to-point queue, but it is possible to implement a fan-out semantic by configure the queue as push
+queue and set other queue as subscribers. More information about that could be found on 
+[IronMq documentation](https://www.iron.io/ironmq-fan-out-support/)
+
+## Artifacts
+
+sbt
+:   @@@vars
+    ```scala
+    libraryDependencies += "com.lightbend.akka" %% "akka-stream-alpakka-ironmq" % "$version$"
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```xml
+    <dependency>
+      <groupId>com.lightbend.akka</groupId>
+      <artifactId>akka-stream-alpakka-ironmq_$scala.binaryVersion$</artifactId>
+      <version>$version$</version>
+    </dependency>
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```gradle
+    dependencies {
+      compile group: "com.lightbend.akka", name: "akka-stream-alpakka-ironmq_$scala.binaryVersion$", version: "$version$"
+    }
+    ```
+    @@@
+
+## Usage
+
+IronMq can be used either in cloud or on-premise. Either way you need a authentication token and a project ID. These can
+be set in the Typesafe config:
+
+: @@snip (../../../../ironmq/src/main/resources/reference.conf)
```

Comment:
Remove the colon before `@@snip ...`, as otherwise the previous paragraph is being treated as a snippet header text.